contact.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" href="portfolio.css">
+	<title>Portfolio Page</title>
+</head>
+<body>
+	<nav >
+		 <div class="name">
+	        <ul>
+				<li>Isabelle Cochrane</li>
+			</ul>
+		</div>
+		<div class="navlinks">
+	<ul>
+	    <li><a href=portfolionav.html>Projects</a></li>
+	    <li><a href=aboutme.html>About Me</a></li>
+	    <li><a href=contact.html>Contact</a></li>
+	 </ul>
+	 </div>
+	</nav>
+	<div>
+		<div class="contactInfo">
+			<h1>CONTACT ME</h1>
+			<div class="icons">
+
+		</div>
+			<div class="general">
+				<h2 class="contact"><a href="https://www.linkedin.com/in/isabelle-cochrane-a97a7314a/"><img src="linkedin.png" alt="linkedin icon"></a></h2>
```

Comment:
The linkedin icon is missing for me, double check that you've got it pushed up to the repo